pkg/webhook/deployment.go

```diff
@@ -15,6 +15,8 @@ package webhook
 import (
 	"context"
 
+	"k8s.io/apimachinery/pkg/util/intstr"
```

Comment:
Tedious nitpick: Could you group the imports in this file? E.g.
```go
import (
  "<library-imports>"

  "<local-repo-imports>"

  "<dependency-imports>"
)
```

Once there's blank lines introduced between imports, editors tend to continue to mangle them.

Reply:
good catch! sorry about that, didn't notice it - fixed here: https://github.com/devfile/devworkspace-operator/pull/309/files#diff-ab898919aca276514502fbe2f2c75745364e965107af8d3f497519260891e271R20